lib/ExponentialBackoff.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var ExponentialBackoff = function(minimum_backoff, exponent, starting_backoff) {
+    if (minimum_backoff === undefined) {
+        minimum_backoff = 50;
+    }
+
+    if (exponent === undefined) {
+        exponent = 2.0;
+    }
+
+    if (starting_backoff === undefined) {
+        starting_backoff = minimum_backoff;
+    }
+
```

Comment:
you could also use the one-liner

```
minimum_backoff = minimum_backoff || 50;
exponent = exponent || 2.0;
starting_backoff = starting_backoff || minimum_backoff;
```

less number of lines of code :smile_cat: